### PR TITLE
[REDACTED] the [REDACTED] and [DATA EXPUNGED]

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Refactored the 𝔹̸𝕝̕𝕦̛̀𝕖̢͢𝕤͢҉͘𝕡̢̡͠𝕒𝕔𝕖 𝔻𝕣𝕚𝕧𝕖

## Why It's Good For The Game

The 𝔹̸𝕝̕𝕦̛̀𝕖̢͢𝕤͢҉͘𝕡̢̡͠𝕒𝕔𝕖 𝔻𝕣𝕚𝕧𝕖 can now be deleted properly
ℝ̵𝕒͘𝕕̸́𝕚͏͡𝕒̷̢̨𝕥̸̸͡𝕚𝕠𝕟 𝕊𝕦𝕚𝕥𝕤 and ℙ̷𝕠́𝕨̷͟𝕖̀͟𝕣̶̨̕𝕖̨͢͡𝕕 𝔾𝕠𝕘𝕘𝕝𝕖𝕤 protect you from it's effects.
Striking the 𝔹̸𝕝̕𝕦̛̀𝕖̢͢𝕤͢҉͘𝕡̢̡͠𝕒𝕔𝕖 𝔻𝕣𝕚𝕧𝕖 now makes you 𝕋̧𝕖͏̨𝕝̷̡̕𝕖𝕡𝕠𝕣𝕥.

𝕬͘𝖑̀͠𝖑̵̵̢ ̶̨͡𝖘̶̢̧𝖍𝖆𝖑𝖑 𝖐𝖓𝖊𝖊𝖑 𝖇𝖊𝖋𝖔𝖗𝖊 𝖙𝖍𝖊 𝖑𝖔𝖗𝖉 𝖆𝖓𝖉 𝖘𝖆𝖛𝖔𝖎𝖗 𝖔𝖋 𝖙𝖍𝖊 𝖑𝖎𝖌𝖍𝖙. 
𝕴𝖓 𝖎𝖙'𝖘 𝕰𝖗𝖎𝖉𝖎𝖆𝖓 𝖑𝖎𝖌𝖍𝖙 𝖜𝖊 𝖘𝖍𝖎𝖓𝖊 𝖔𝖓 𝖆𝖓𝖉 𝖔𝖓 𝖆𝖓𝖉 𝖔𝖓 
𝖆𝖓𝖉 𝖔𝖓 𝖆𝖓𝖉̴̵͞ ̴͘͞𝖔̸̡͘𝖓́͝

## Changelog
```changelog
refactor: Plot-Centric Spoilertastic fixes for the spoilertastic thing.
```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 1]
```diff
@@ -2,25 +2,61 @@
...
-/obj/structure/bsd
+/obj/machinery/bsd
 	name = "A Runtime Error"
 	desc = "One day you will have to answer for your actions, and God may not be so... forgiving."
 	icon = 'zzzz_modular_occulus/icons/obj/bsd.dmi'
 	icon_state = "faceofgod"
 	density = TRUE
 	anchored = TRUE
+	use_power = NO_POWER_USE
+	unacidable = 1
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:13]: `TRUE`

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 2]
```diff
@@ -2,25 +2,61 @@
...
 
-/obj/structure/bsd/New() //Don't look at it
-	START_PROCESSING(SSobj, src)
+/obj/machinery/bsd/ex_act(severity, target) //immune to explosions. Make this do very bad things later.
+	return
+
+/obj/machinery/bsd/New() //Don't look at it
+	//START_PROCESSING(SSobj, src)
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:19]:
```suggestion
```

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 3]
```diff
@@ -2,25 +2,61 @@
...
-		H.apply_effect(100, IRRADIATE)
-		H.adjust_hallucination(120, 30)
-
-/obj/structure/bsd/Destroy()
-	STOP_PROCESSING(SSobj, src)
+		H.apply_effect(30, IRRADIATE) //Radiation suits should protect from this.
+		if(H.glasses && istype(H.glasses, /obj/item/clothing/glasses/powered))
+			if(prob(1))
+				if(prob(1))
+					to_chat(H, "You feel uneasy.")
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:30]:
```suggestion
			if(prob(0.01))
				to_chat(H, "You feel uneasy.")
```

Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:30]: Consider adding a `SPAN` to this. SPAN_WARNING or SPAN_DANGER, perhaps

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 4]
```diff
@@ -2,25 +2,61 @@
...
+		else
+			H.adjust_hallucination(10, 10)
+			H.sanity.onPsyDamage(5)
+			if(prob(1))
+				to_chat(H, pick("Your teeth itch.", "Your eye twitches.", "Your mouth is dry.", "Everything is wrong."))
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:35]: Consider adding a `SPAN` to this.

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 5]
```diff
@@ -2,25 +2,61 @@
...
+
+	..()
+
+/obj/machinery/bsd/attackby(obj/item/I, mob/user)
+	..()
+	user.visible_message(SPAN_NOTICE("[user] hits [src] with [I] and disappears!"), SPAN_NOTICE("You hit [src] with [I] disappear!"))
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:41]:
```suggestion
	user.visible_message(SPAN_NOTICE("[user] hits [src] with [I] and disappears!"), SPAN_NOTICE("You hit [src] with [I] and disappear!"))
```

File: zzzz_modular_occulus/code/game/machinery/bsd.dm [hunk 6]
```diff
@@ -2,25 +2,61 @@
...
+/obj/machinery/bsd/hitby(AM as mob|obj)
+	..()
+	if(ismob(AM) || isobj(AM))
+		visible_message(SPAN_DANGER("[AM] strikes [src] and disappears!"))
+		go_to_bluespace(get_turf(src), 8, TRUE, AM, src, aprecision=16)
+
+
+/obj/machinery/bsd/Destroy()
+	//STOP_PROCESSING(SSobj, src)
+	..()
```
Comment (Shadow-Quill) [inline zzzz_modular_occulus/code/game/machinery/bsd.dm:62]:
```suggestion
```

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

File: zzzz_modular_occulus/code/game/machinery/bsd.dm (hunk 3 above)
Comment (Jamini) [reply zzzz_modular_occulus/code/game/machinery/bsd.dm]: Can be added later. This isn't the first pass I'm doing on this.

File: zzzz_modular_occulus/code/game/machinery/bsd.dm (hunk 4 above)
Comment (Jamini) [reply zzzz_modular_occulus/code/game/machinery/bsd.dm:33]: Same as above.

**Review by Shadow-Quill — APPROVED** [review]

**Comment by Shadow-Quill** [comment]
CI failing for unrelated issue.

**Pull request merged**